package.json

```diff
@@ -2,7 +2,6 @@
   "name": "lunes",
   "version": "0.0.1",
   "private": true,
-  "packageManager": "yarn@3.2.2",
```

Comment:
Also why do we not need this field anymore? Sorry if it's a stupid question, but I could not find much information about it online